packages/common/core-utils/package.json

```diff
@@ -52,9 +84,11 @@
 		"test": "npm run test:mocha",
 		"test:benchmark:report": "mocha --node-option unhandled-rejections=strict,expose-gc --exit --perfMode --fgrep @Benchmark --reporter @fluid-tools/benchmark/dist/MochaReporter.js --timeout 60000",
```

Comment:
I think this needs updated if the spec is to be kept removed from the config file.

Reply:
You're right.  Good catch!